.coveragerc

```diff
@@ -255,9 +255,7 @@ omit =
     homeassistant/components/dormakaba_dkey/sensor.py
     homeassistant/components/dovado/*
     homeassistant/components/downloader/__init__.py
-    homeassistant/components/dsmr_reader/__init__.py
     homeassistant/components/dsmr_reader/definitions.py
```

Comment:
Can `definitions.py` not also be removed from `.coveragerc`?

Reply:
The scope is the sensor, for now.

Reply:
Yes, but definitions is just static constants imported into the sensor.
I would expect these to be automatically tested as part of the sensor tests, same as `__init__.py`

Reply:
Well, it has just been added. Going for full coverage now.